Search2dMatrix.java

```diff
@@ -0,0 +1,38 @@
+// Time complexity is: O(Log n)
```

Comment:
TC is wrong

Reply:
it should be O (Log mn) because its a 2D matrix